app/models/variant_override.rb

```diff
@@ -21,6 +19,12 @@ class VariantOverride < ActiveRecord::Base
     where(hub_id: hubs)
   }
 
+  scope :distinct_import_dates, lambda {
+    select('DISTINCT variant_overrides.import_date').
```

Comment:
A pity that we're still on Rails 3.2 otherwise we could make use of AR's [distinct](https://apidock.com/rails/v4.0.2/ActiveRecord/QueryMethods/distinct)